base/gfspclient/signer.go

```diff
@@ -78,6 +78,30 @@ func (s *GfSpClient) SealObject(ctx context.Context, object *storagetypes.MsgSea
 	return nil
 }
 
+func (s *GfSpClient) RejectUnSealObject(
+	ctx context.Context,
+	object *storagetypes.MsgRejectSealObject) error {
```

Comment:
these parameters can be moved to one line.

Reply:
fixed